go.mod

```diff
@@ -67,15 +69,14 @@ require (
 	golang.org/x/exp v0.0.0-20240213143201-ec583247a57a // indirect
 	golang.org/x/mod v0.17.0 // indirect
 	golang.org/x/net v0.34.0 // indirect
-	golang.org/x/oauth2 v0.17.0 // indirect
+	golang.org/x/oauth2 v0.27.0 // indirect
```

Comment:
@tmshort 

I need your help with this one we need use golang 1.23 to bump
I create a PR for opc/release: https://github.com/openshift/release/pull/62315
Could you help within? 
I think after that get merged it should pass in the tests right?